docs/source/jupyter.rst

```diff
@@ -7,3 +7,4 @@ Jupyter notebooks
    /examples/Symbolically Understanding Christoffel Symbol and Riemann Curvature Tensor using EinsteinPy.ipynb
    /examples/Analysing Earth using EinsteinPy!.ipynb
    /examples/Visualizing frame dragging in Kerr spacetime.ipynb
+   /examples/Visualizing event horizon and ergosphere of Kerr black hole.ipynb
```

Comment:
Add the notebook's best plot/image as `kerrblackhole.png` in the same examples folder.

Reply:
I'll do it